keras/src/layers/normalization/batch_normalization.py

```diff
@@ -314,39 +314,44 @@ def _moments(self, inputs, mask):
         if mask is None:
             return ops.moments(
                 inputs,
-                axes=self._reduction_axes,
+                axes=self._reduction_axes
+                if isinstance(self._reduction_axes, list)
```

Comment:
Can we normalize `self._reduction_axes` to a list earlier (when we create the attr)?